src/Object/Clarkson_Object.php

```diff
@@ -416,12 +416,10 @@ public function get_excerpt() {
 	}
 
 	/**
-	 * Get the post's post type.
-	 *
-	 * @return string
+	 * Get the post's post type object.
 	 */
-	public function get_post_type() {
-		return $this->_post->post_type;
+	public function get_post_type(): ?Clarkson_Post_Type {
+		return Clarkson_Post_Type::get( $this->_post->post_type );
```

Comment:
Ik denk dat het handiger is om hiervoor een extra functie aan te maken, zodat de oude functie get_post_type() kan blijven bestaan.
Vaak wil je denk ik niet het hele object, maar gewoon de naam.

Reply:
Dan volstaat op zich `$clarkson_object->get_post()->post_type`?

Reply:
Dat klopt, maar we willen toch juist niet dat mensen direct properties uit het post object gaan opvragen?

Reply:
Het is niet perse de bedoeling om voor alles waar WordPress al een redelijke API hier een soort alias voor te maken wat mij betreft.

Reply:
Maar dan komt vraag 2: what to do met alle functies die nu al bestaan om directe object properties op te halen?
Wat wil je dan met een functie als `get_status()`?

Reply:
Wat denk jij dat handig is?

Reply:
Ik zou graag een vorm van consistentie hierin zien, dat vind ik het meest belangrijk.
Dus of overal 
`$clarkson_object->get_post()->post_status` of overal `$clarkson_object->get_status()`.

Ik denk zelf dat het fijner is om hiervoor gewoon de functie die we nu hebben, `$clarkson_object->get_status()`, te behouden. Maakt je code op veel plekken gewoon een stuk korter.
Dan zou ik er dus ook fan van zijn om de `get_post_type()` functie terug te zetten naar hoe die was, en een extra functie te maken genaamd `get_post_type_object()` of iets dergelijks, die doet wat je functie nu doet.